user/tests/wiring/ble_central_peripheral/ble_central/central.cpp

```diff
@@ -540,5 +541,28 @@ test(BLE_31_Pairing_Receiption_Reject) {
     }
 }
 
+test(BLE_32_Att_Mtu_Exchange) {
+#define LOCAL_DESIRED_ATT_MTU 100
+#define PEER_DESIRED_ATT_MTU 123
+
+    BLE.onAttMtuExchanged([&](const BlePeerDevice& p, size_t attMtu) {
```

Comment:
This is a capturing lambda, might be a good idea to reset the callback after exiting the test.